microsoft/PowerToys
# [FancyZones] Increase the number of zones limit to 128

## Summary of the Pull Request

**What is this about:**

* Added property to save zone number up to 128 
* Updated zone limit constant in the editor.

**What is included in the PR:** 

**How does someone test / validate:** 

* Apply a template layout with >40 zones
* Verify correct `zone-count` in the `layout-templates.json`
* Snap window, verify correct `zone-index-set` in the `app-zone-history.json`

## Quality Checklist

- [x] **Linked issue:** #7051

## Contributor License Agreement (CLA)
A CLA must be signed. If not, go over [here](https://cla.opensource.microsoft.com/microsoft/PowerToys) and sign the CLA.
